assignments/closure.js

```diff
@@ -1,19 +1,36 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+let simple = () => {let x = 5};
 
```

Comment:
I'm not sure how clearly this demonstrates that you understand closure. I can see how you could convince me, but without having that conversation this isn't super explicit. Josh's example was this:
```
let myName = "Josh";

function sayHello() {
  let example = "jim";
  // accessing myName outside this function
  return `Hello ${myName}`;
}
console.log(sayHello());

console.log(example);
```
Note how he defines the variable outside the scope of the function, but calls it from within. There are also variables defined within the function that we could talk about not being able to be called from outside. Just a bit more explicit in its demonstration of closure.